### ISSUE TITLE
Protect main and develop branches

### ISSUE THREAD

**Issue by maximekuhn** [Meliodas-1O/modernMoodle#40]
Protect `main` and `develop` branches to prevent force push on them.

Maybe it has already been made ? @Meliodas-1O

**Comment by Meliodas-1O** [comment]
Yes, it would be a great idea as Github recommends it to me every time I log in

**Comment by maximekuhn** [comment]
I'am assigning you on this task since you own the repo !
